CourseGrab/ViewControllers/Search/SearchTableViewController.swift

```diff
@@ -89,6 +92,7 @@ extension SearchTableViewController {
                     self.tableView.reloadData()
                 }
             }
+            timer?.invalidate()
```

Comment:
what's this timer for 👀

Reply:
sry for the terribly late reply. I missed this. 
So basically we have a timer of 0.2 seconds to send the search request to the backend. This gives the user a little time we type more text before we send the request. Otherwise, we send too many search requests to the backend. The invalidate part just invalidates the last timer so we don't send the last request.